google/cloud/forseti/notifier/notifiers/inventory_summary.py

```diff
@@ -98,6 +102,10 @@ def _upload_to_gcs(self, summary_data):
                     self._get_output_filename(
                         string_formats.INVENTORY_SUMMARY_JSON_FMT))
                 file_uploader.upload_json(summary_data, gcs_upload_path)
+            log_message = f'Inventory Summary file saved to GCS path: ' \
+                          f'{gcs_upload_path}'
+            self.progress_queue.put(log_message)
```

Comment:
Main change - log the path of the GCS file to the progress queue (i.e. CLI) so that automated test can parse this path and verify the file exists.